Assets/_BForBoss/_Character/Scripts/FirstPersonPlayer.cs

```diff
@@ -90,14 +90,20 @@ protected override void HandleInput()
         protected override void OnOnDisable()
         {
             base.OnOnDisable();
-            _dashBehaviour.OnOnDisable();
+            if (_dashBehaviour != null)
+            {
+                _dashBehaviour.OnOnDisable();
+            }
```

Comment:
I forget, is there a reason we don't use null propagation (i.e. `_dashBehaviour?.OnDisable()`) for Monobehaviours? I remember there was reasoning but I forgot what it was!